truckersmp_cli/configfile.py

```diff
@@ -28,6 +29,34 @@ def __init__(self, configfile):
             # get data from valid thirdparty.* sections
             if not sect.startswith("thirdparty.") or "executable" not in parser[sect]:
                 continue
+            # only use configurations for the specified game
+            #   [thirdparty.prog1]        -> all games
+            #   [thirdparty.ets2mp.prog1] -> ETS2MP
+            #   [thirdparty.ets2.prog1]   -> ETS2
+            #   [thirdparty.atsmp.prog1]  -> ATSMP
+            #   [thirdparty.ats.prog1]    -> ATS
+            if Args.singleplayer:
+                if Args.ets2:  # ets2
+                    if (sect.startswith("thirdparty.ets2mp.")
+                            or sect.startswith("thirdparty.ats.")
+                            or sect.startswith("thirdparty.atsmp.")):
+                        continue
+                else:          # ats
+                    if (sect.startswith("thirdparty.ets2mp.")
+                            or sect.startswith("thirdparty.ets2.")
+                            or sect.startswith("thirdparty.atsmp.")):
+                        continue
+            else:
+                if Args.ets2:  # ets2mp
+                    if (sect.startswith("thirdparty.ets2.")
+                            or sect.startswith("thirdparty.ats.")
+                            or sect.startswith("thirdparty.atsmp.")):
+                        continue
+                else:          # atsmp
+                    if (sect.startswith("thirdparty.ets2mp.")
+                            or sect.startswith("thirdparty.ets2.")
+                            or sect.startswith("thirdparty.ats.")):
+                        continue
```

Comment:
This section looks overly complicated.

What about something like
```suggestion
            # get data from valid thirdparty.* sections
            # only use configurations for the specified game
            #   [thirdparty.all.prog1]    -> all games
            #   [thirdparty.ets2mp.prog1] -> ETS2MP
            #   [thirdparty.ets2.prog1]   -> ETS2
            #   [thirdparty.atsmp.prog1]  -> ATSMP
            #   [thirdparty.ats.prog1]    -> ATS
            if not sect.startswith("thirdparty.all") or not sect.startswith("thirdparty." + Args.game) or "executable" not in parser[sect]:
                continue
```